libs/extended/src/System/Logger/Extended.hs

```diff
@@ -23,3 +28,21 @@ mkLogger' = Log.new
     . Log.setOutput Log.StdOut
     . Log.setFormat Nothing
     $ Log.defSettings
+
+newtype LoggerT m a = LoggerT {runLoggerT :: ReaderT Log.Logger m a}
+    deriving newtype
+        (Functor
+        , Applicative
+        , Monad
+        , MonadIO
+        , MonadThrow
+        , MonadCatch
+        , MonadMask
+        , MonadClient
+        )
+
+instance (MonadIO m) => LC.MonadLogger (LoggerT m) where
```

Comment:
Should this be overlappable?  Or perhaps it's better not to add that now, and wait for the overlapping situation to arise.